src/psij/web/css/style.css

```diff
@@ -154,38 +154,68 @@ body {
 }
 
 #breadcrumbs {
-    float: left;
+    position: absolute;
+    top: 33px;
+    background-color: #444444;
+    border: 1px solid #666666;
+    z-index: 2;
+}
+
+#breadcrumbs .v-breadcrumbs__divider {
+    color: white;
 }
 
 #breadcrumbs .v-breadcrumbs {
     margin: 0pt;
-    padding-bottom: 4pt;
-    padding-top: 4pt;
+    padding-bottom: 0;
+    padding-top: 0;
 }
 
 #breadcrumbs li:not(.v-breadcrumbs__divider) {
     display: inline-block;
-    box-shadow: 1pt 1pt 3pt #606060;
     cursor: pointer;
-    background-color: #ffffff;
-    padding: 4pt;
+    padding: 1pt;
+    color: white;
+}
+
+#breadcrumbs li {
+    font-size: 12px;
+}
+
+#breadcrumbs a:visited {
+    color: white;
 }
 
 #breadcrumbs li:hover:not(.v-breadcrumbs__divider) {
-    box-shadow: 1pt 1pt 5pt #80a0ff !important;
+    color: white;
 }
 
+.dashboard-title {
+    position: absolute;
+    left: 15px;
+    top: 6px;
+    color: #d0d0d0;
+    font-size: 14px;
+    font-weight: bold;
+}
 
 #toolbar .button {
     display: inline-block;
-    box-shadow: 1pt 1pt 3pt #606060;
     cursor: pointer;
-    background-color: #ffffff;
-    padding: 3pt;
+    background-color: #b2b2b2;
+    padding: 5px 9px 6px 9px;
+    border-top: 1px solid black;
+    border-left: 1px solid black;
+    border-right: 1px solid black;
+    border-bottom: 1px solid black;
+    margin: 0 5px 0 0;
+    border-top-right-radius: 5px;
+    border-top-left-radius: 5px;
```

Comment:
Think of the view modes more like the "view as list" / "view as icons" in windows explorer and Finder. They are not tabs, since the information doesn't change. What changes is how the same information is represented.